app/src/main/java/org/ole/planet/myplanet/ui/course/CourseFragment.java

```diff
@@ -90,6 +91,9 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
         });
         clearTags();
         showNoData(tvMessage, adapterCourses.getItemCount());
+        keyboardUtils = new KeyboardUtils();
+        keyboardUtils.setupUI(getView().findViewById(R.id.my_course_parent_layout),getActivity());
+
```

Comment:
@ChiragAwale  change setupUI to static, as it is utility function, so you don't have to create object every time you wants to call this method.

Reply:
Got it thanks :D

Reply:
@rrijal53 Please look at the changes :D